### ISSUE TITLE
Log Failure with 4.5

### ISSUE THREAD

**Issue by branic** [redhat-cfme-support/cloudforms_4.X_log_collection#8]
Collecting the logs on a 4.5 appliance that is not the VMDB appliance fails with the following error:

`
tar: /var/opt/rh/rh-postgresql95/lib/pgsql/data/pg_log/*: Cannot stat: No such file or directory
tar: /var/opt/rh/rh-postgresql95/lib/pgsql/data/postgresql.conf: Cannot stat: No such file or directory
tar: Exiting with failure status due to previous errors
`

The error appears to be with the check to determine if the appliance is the VMDB. The environment variable $APPALIANCE_PG_DATA is being checked to see if it is defined. In all of the CloudForms 4.5 appliances that I've checked that environment variable is set regardless of whether the appliance has the VMDB or not.

**Comment by jeffmcutter** [comment]
+1 - In my case seems to get everything else OK but is confusing.

**Comment by jeffmcutter** [comment]
Submitted PR #9 to fix this.

**Comment by bmduffy** [comment]
Hi, if $APPALIANCE_PG_DATA isn't defined you get an empty string which doesn't fail the directory test. The script needs to check for an empty string in this case.

Submitted PR #10 to fix this.

**Comment by jeffmcutter** [comment]
I believe that PR #10 is based upon a false assumption, at least for 4.5 appliances.  $APPLIANCE_PG_DATA is defined on all 4.5 CF appliances, and the directory also exists.  Also, I tested the merged #10 and it finds that all CFME servers are non-database even when they are.  This is due to checking if if $APPLIANCE_PG_DATA is zero instead of non-zero.  I have submitted PR #11 to address these concerns.
